voltixApp/VoltixApp/Services/RpcEvmService.swift

```diff
@@ -105,8 +104,17 @@ class RpcEvmService {
         return try await intRpcCall(method: "eth_getBalance", params: [address, "latest"])
     }
     
-    private func fetchMaxPriorityFeePerGas() async throws -> BigInt {
-        return try await intRpcCall(method: "eth_maxPriorityFeePerGas", params: [])
+    func fetchMaxPriorityFeePerGas() async throws -> BigInt {
+        let feeInGwei = try await intRpcCall(method: "eth_maxPriorityFeePerGas", params: [])
+        
+        let feeInGweiString = feeInGwei.description
```

Comment:
I think this is wrong , the result return from the RPC is in Wei , so we always need to convert it to GWei here , and also we should maybe keep the minimum as 1 Gwei

Reply:
**BSC:**
priorityFeeGwei: 1000000000
TICKER: USDT - BSC
gas: 1
nonce: 2

priorityFeeGwei: 1000000000
TICKER: BNB
gas: 1
nonce: 2

**AVAX**

TICKER: AVAX
gas: 25
nonce: 6
priorityFeeGwei: 0

TICKER: USDC - AVAX
gas: 25
nonce: 6
priorityFeeGwei: 0

**Ethereum**

TICKER: ETH
gas: 12
nonce: 2
priorityFeeGwei: 1000000

TICKER: USDC
gas: 13
nonce: 2
priorityFeeGwei: 970000

Reply:
Ethereum

TICKER: ETH
gas: 12
nonce: 2
priorityFeeGwei: 1000000

TICKER: USDC
gas: 13
nonce: 2
priorityFeeGwei: 970000

This means ETH's priority fee is less then 1 gwei , then we can just use 1gwei

Reply:
Yep. I've done that.